### PR TITLE
Task 3.1

### PR DESCRIPTION
**Pull request by akravchenko564**

### CODE REVIEW

File: Practicum/JSBook/Z3.1.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿function ucFirst(str) {
+	return str[0].toUpperCase() + str.substring(1);
```
Comment (KasichMax) [inline Practicum/JSBook/Z3.1.js]: What if str is empty?

File: Practicum/JSBook/Z3.1.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+}
+alert (ucFirst('vasya'));
```
Comment (KasichMax) [inline Practicum/JSBook/Z3.1.js]: Add `alert(ucFirst(""));` ?

File: Practicum/JSBook/Z3.1.js (hunk 1 above)
Comment (akravchenko564) [inline Practicum/JSBook/Z3.1.js]: Returns nothing

File: Practicum/JSBook/Z3.1.js (hunk 2 above)
Comment (akravchenko564) [inline Practicum/JSBook/Z3.1.js]: When added `alert(ucFirst(""));` it's return only first alert: `alert (ucFirst('vasya'));` => Vasya.

File: Practicum/JSBook/Z3.1.js (hunk 1 above)
Comment (KasichMax) [inline Practicum/JSBook/Z3.1.js]: Maybe you have an error in console tab?
You should handle case with empty string (read a task one more time).

File: Practicum/JSBook/Z3.1.js (hunk 2 above)
Comment (KasichMax) [inline Practicum/JSBook/Z3.1.js]: See my previous comment. You receive an error.

File: Practicum/JSBook/Z3.1.js (hunk 1 above)
Comment (akravchenko564) [inline Practicum/JSBook/Z3.1.js]: About what srt you told?

Comment (akravchenko564) [inline Practicum/JSBook/Z3.1.js]: Done ac410c9e53175509a4a593402bb3f74c720c5049

**Pull request merged**